### ISSUE TITLE
interact creditfraud issue - segmentation fault.

### ISSUE THREAD

**Issue by Hobstarr** [Azure/counterfit#35]
Trying to use some of the demos on counterfit, but running into a problem when trying:

-----------------
interact creditfraud.
zsh: segmentation fault python counterfit.py
-----------------

interact works for digits_blackbox, digits_keras, movie_reviews and satellite. Just specifically a problem with creditfraud.

**Comment by moohax** [comment]
Interesting. Have a stacktrace or anything I can look at?

**Comment by Hobstarr** [comment]
> Interesting. Have a stacktrace or anything I can look at?

I'm not exactly sure how to do that. Happy to do it if you can explain how. :)

**Comment by Hobstarr** [comment]
> Interesting. Have a stacktrace or anything I can look at?

Hi sorry, I did reply to this but not directly. I'd be happy to do this if you explain how.

**Comment by moohax** [comment]
Right, of course, just says "segmentation fault". So the only thing interact does is call the `target.load()` function. Can you load the model manually in a simple Python interpreter? The data too?
